### PR TITLE
Restored functionality of the login/logout links.

### PR DESCRIPTION
**Pull request by adelevie**
- Restored the login/logout buttons at the top of the layout
- Added a `/login` route per @blacktm's UI sketches (`LoginsController#index`)

### CODE REVIEW

**Comment by adelevie** [comment]
Fixes https://github.com/18F/micropurchase/issues/51 and https://github.com/18F/micropurchase/issues/50.

File: app/assets/javascripts/logins.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```
Comment (baccigalupi) [inline app/assets/javascripts/logins.js]: Can we can probably get rid of these generated empty files?

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  #protect_from_forgery with: :exception
```
Comment (baccigalupi) [inline app/controllers/application_controller.rb]: I think we probably want to keep this. Was this commented out for debugging, or is there another issue going on?

File: app/helpers/logins_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module LoginsHelper
+end
```
Comment (baccigalupi) [inline app/helpers/logins_helper.rb]: This too can probably go

File: app/assets/javascripts/logins.js (hunk 1 above)
Comment (adelevie) [inline app/assets/javascripts/logins.js]: Yup!

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (adelevie) [inline app/controllers/application_controller.rb]: good catch ... this was an artifact of debugging.

File: app/helpers/logins_helper.rb (hunk 3 above)
Comment (adelevie) [inline app/helpers/logins_helper.rb]: on it.

File: app/views/users/confirm.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<h2>Complete your Account</h2>
```
Comment (baccigalupi) [inline app/views/users/confirm.html.erb]: users#edit is probably a better REST representation.

File: spec/helpers/logins_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
```
Comment (baccigalupi) [inline spec/helpers/logins_helper_spec.rb]: I would ditch this file one too.

**Comment by adelevie** [comment]
rebased

**Pull request merged**